go.mod

```diff
@@ -38,7 +38,7 @@ require (
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
-	go.opentelemetry.io/collector v0.38.0
+	go.opentelemetry.io/collector v0.39.0
```

Comment:
Don't bump up the version in this PR. It'll break things if it's only partially done. That's what https://github.com/aws-observability/aws-otel-collector/pull/750 is for.

Reply:
Fixed it in the new commit but forgot to push it. Thanks for reminding me.